civil.md

```diff
@@ -34,24 +36,36 @@ Before we talk about how to name them, we should be clear on what things we are
 
+**Con:** Despite appearances, the date class conceptually represents a date irrespective of any calendar system. You only need one representation of a date in the domain model, with calendar system output a localization issue. The date class exposes year/month/day because it is convenient for the vast majority of use cases to do so.
+
 **Con:** Gregorian requires knowledge of the fact that the *proleptic gregorian calendar* is what most of the western world uses. This is not knowledge we can assume.
 
 **Con:** `CivilTime` does not in any way depend on the gregorian calendar. In fact it is a secondary characteristic: *the midnight to midnight* day definition. Even in cultures that use the gregorian calendar, this definition may not apply (think Ethiopia). So gregorian would be factually wrong for times and misleading in combination with some calendars.
 
 ## Local
 
-**Pro:** Local is the prefix that many programming languages use for this type of object. It is therefore very intuitive to many users.
+**Pro:** Local is the prefix that many programming languages use for exactly these types of object for around 20 years. It is therefore very intuitive to many users and widely defined in internet searches and help forums.
+
+**Pro:** ISO-8601 used "local time" over 20 years ago for the concept:
+
+> 2.1.16 local time: locally applicable time of day such as standard time of day, or a non-UTC based time of day"
 
-**Con:** Local is actually wrong in that it names the one characteristic that these objects do **not** possess: locality.
+The key is that it is "locally applicable", not that it implies a location.
 
-**Con:** Local is the prefix that many languages use. However the `Civil` objects are part of a larger whole with a very specific mental model. Local violates that mental model and by suggesting analogies to other languages it exacerbates that misunderstanding of the meaning and value of these types.
+**Pro:** The term "local time" is in widespread use, both in scholarly articles and common parlance. For example, airline pilots will typically say "when we land the local time will be 7pm".
+
+**Con:** Local names a characteristic that these objects do not possess: locality. But surely this misunderstands how "local" is used in English - as with the airline case, the phrase "local time" doesn't refer to knowledge of the locality, but the absence of needing to worry about it on your watch/phone once it is set correctly. As a related example, talking about your "local shop" doesn't require identifying a specific shop in a specific locality. Instead it is a phrase used to discuss whatever the shop nearest to you is, something that varies depending on where you are.
```

Comment:
The language "But surely this misunderstands" is too strong.  Please soften up the new material you added in this bullet.